vm/boolean.go

```diff
@@ -39,6 +39,10 @@ func (b *BooleanObject) ReturnClass() Class {
 	return b.Class
 }
 
+func initializeBoolean(value bool) *BooleanObject {
```

Comment:
We should not have a method to initialize boolean. I already use variable `TRUE` and `FALSE` to represent true and false. Since they won't be modified, we can share them between the program without creating new one.